driver/postgresql/driver.go

```diff
@@ -40,8 +41,16 @@ type Conn struct {
 
 // HasTable check if the schema has the migration table.
 func (c *Conn) HasTable() (bool, error) {
+	pgSchema := "public"
```

Comment:
I think `schema` is enough here. We're in the postgresql driver, it's not going to be anything else. Same for  `pgTable`.

Reply:
I don't follow. Your use of schema == pg database. My use of pgSchema == pg schema. In PG, there is a differentiation between schemas and databases, which is lacking in MySQL. The focus of the PR is to make this distinction implicit as you requested in an earlier PR rather than making it explicit in the configuration value.

I guess this is as good a place to ask as any. Why doesn't the Conn implement the initialization logic to create the sql.DB? It's in the Service right now. Would you be open to a change that passes the Env to the Conn's to allow them to get the database connection? I can add a Schema option to the env. A MySQL user could use either RAMBLER_DATABASE or RAMBLER_SCHEMA. PG would default to "public" if RAMBLER_SCHEMA is empty.

Reply:
The driver itself doesn't implement the initialization logic because it wasn't necessary when I first wrote the software, and the later addition of the PostgreSQL support wasn't enough for me to want to spend time refactoring it all. I merely asked for the author of the driver to keep the compatibility, which was done. 

I never used PostgreSQL myself, which is why I didn't see the inconsistency with the variable naming. Note that you're free to refactor the whole driver as you want to fix this, but please refrain from changing the interface itself to add something that is PostgreSQL-specific and forcing everyone to follow the same convention. PostgreSQL is the special case here, not the other drivers.